From 0b05ff5b71f230cd90ac7ae136533f05e17fc57e Mon Sep 17 00:00:00 2001
From: "zongren.yin" <zongren.yin@amlogic.com>
Date: Mon, 11 Dec 2023 10:10:45 +0800
Subject: [PATCH 1/2] smarthome: machine config update [1/1]

PD#SWPL-149327

Problem:
S4 lost aiot common header;

Solution:
Add common header to S4 config;
Move video convert to encoder_common

Fixed conflicts:
NA

Verify:
local

Signed-off-by: zongren.yin <zongren.yin@amlogic.com>
Change-Id: I39c2bb41e989b6cfbb8ad4fd8f9f8fc466a09817
---
 conf/machine/include/amlogic_aiot_common.inc  | 1 -
 conf/machine/include/amlogic_enc_common.inc   | 1 +
 conf/machine/mesons4-ap222-weston-5.15.config | 1 +
 3 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/conf/machine/include/amlogic_aiot_common.inc b/conf/machine/include/amlogic_aiot_common.inc
index 8eb760d1..6502195b 100644
--- a/conf/machine/include/amlogic_aiot_common.inc
+++ b/conf/machine/include/amlogic_aiot_common.inc
@@ -9,7 +9,6 @@ DISTRO_FEATURES_append = " gst-aml-gfx2d"
 DISTRO_FEATURES_append = " dma-allocator"
 DISTRO_FEATURES_append = " gst-aml-dma-allocator"
 
-DISTRO_FEATURES_append = " gst-plugin-conv"
 DISTRO_FEATURES_append = " vendor-image"
 
 
diff --git a/conf/machine/include/amlogic_enc_common.inc b/conf/machine/include/amlogic_enc_common.inc
index 4cd4e667..af0e1b57 100644
--- a/conf/machine/include/amlogic_enc_common.inc
+++ b/conf/machine/include/amlogic_enc_common.inc
@@ -5,5 +5,6 @@
 
 CONNECTIVITY_CHECK_URIS = "https://www.yoctoproject.org/"
 
+DISTRO_FEATURES_append = " gst-plugin-conv"
 DISTRO_FEATURES_append = " gst-aml-videotranscoding"
 DISTRO_FEATURES_append = " videotranscoding-demo "
\ No newline at end of file
diff --git a/conf/machine/mesons4-ap222-weston-5.15.config b/conf/machine/mesons4-ap222-weston-5.15.config
index b1ee5ee2..eae6fe67 100644
--- a/conf/machine/mesons4-ap222-weston-5.15.config
+++ b/conf/machine/mesons4-ap222-weston-5.15.config
@@ -5,6 +5,7 @@
 
 include conf/machine/include/mesons4_k5.15_64b.inc
 include conf/machine/include/amlogic_weston.inc
+include conf/machine/include/amlogic_aiot_common.inc
 
 PREFERRED_PROVIDER_virtual/mesa = "libgles-eabihf-dvalin-wayland-drm"
 PREFERRED_PROVIDER_virtual/mesa-gl = "libgles-eabihf-dvalin-wayland-drm"
-- 
2.25.1

